src/cryptonote_core/cryptonote_tx_utils.cpp

```diff
@@ -579,19 +600,20 @@ bool construct_tx_and_get_tx_key(const account_keys &sender_account_keys, const
 			additional_tx_keys.push_back(keypair::generate(sender_account_keys.get_device()).sec);
 	}
 
-	bool r = construct_tx_with_tx_key(sender_account_keys, subaddresses, sources, destinations, change_addr, extra, tx, unlock_time, tx_key, additional_tx_keys, rct, bulletproof, msout);
+	bool r = construct_tx_with_tx_key(sender_account_keys, subaddresses, sources, destinations, change_addr, 
+									  payment_id, tx, unlock_time, tx_key, additional_tx_keys, bulletproof, msout, use_uniform_pids);
```

Comment:
note: this line mixes spaces and tabs. Depending on the configured tab size it looks ugly.
If we indent by tabs we should not try to align arguments. Instead we should use only one indention on the next line